ui/pages/index.tsx

```diff
@@ -82,7 +82,7 @@ export default function Index() {
             <p>
               Provide liquidity in the{' '}
               <a
-                href="https://app.balancer.fi/#/pool/0x0bf37157d30dfe6f56757dcadff01aed83b08cd600020000000000000000019a"
+                href={`${balancerDomain}/#/pool/0x0bf37157d30dfe6f56757dcadff01aed83b08cd600020000000000000000019a`}
```

Comment:
Also replace the mainnet pool ID (`0x0bf37157d30dfe6f56757dcadff01aed83b08cd600020000000000000000019a`) with a variable, since the pool ID for Goerli is different.